### PR TITLE
CE-RKE2-updates-3.16

### PR DESCRIPTION
**Pull request by bmckercher123**
### RKE2 updates for 3.16
https://deploy-preview-378--tigera.netlify.app/calico-enterprise/3.16/getting-started/install-on-clusters/rke2

- Update version per QE to 2.6.5 and later (CC and CE)
- Update RKE2 product name (exposed only in CE)
- CE 3.16/next
- CC RKE2 version only, 3.16

### CODE REVIEW

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx [hunk 1]
```diff
@@ -2,61 +2,62 @@
...
-The geeky details of what you get:
+   The geeky details of what you get:
 
-<GeekDetails
-  prodname='{{prodname}}'
-  details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes'
-/>
+   <GeekDetails
+    prodname='{{prodname}}'
+    details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:BGP,Datastore:Kubernetes'
```
Comment (sujeet-kr) [inline calico-enterprise/getting-started/install-on-clusters/rke2.mdx]:
```suggestion
    details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes'
```

File: calico-cloud/get-started/connect/requirements/rke2.mdx [hunk 2]
```diff
@@ -14,7 +14,7 @@ Although {{prodname}} checks that your cluster meets [System requirements](syste
 
 ## Verify RKE2 settings
 
-1. Verify you have a compatible [RKE2 cluster](https://docs.rke2.io/).
+1. Verify you have a compatible [RKE2 cluster](https://docs.rke2.io/) 2.6.5 and later
```
Comment (sujeet-kr) [inline calico-cloud/get-started/connect/requirements/rke2.mdx]:
```suggestion
1. Verify you have a compatible [RKE2 cluster](https://docs.rke2.io/)
```
The version 2.6.5 is related to Rancher and not RKE2.

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx [hunk 3]
```diff
@@ -2,61 +2,62 @@
...
+   />
 
 **Required**
 
-- A [compatible RKE2 cluster](../compatibility.mdx#rke2)
-
+- A [compatible RKE2 cluster](../compatibility.mdx#rke2) 2.6.5 and later
```
Comment (sujeet-kr) [inline calico-enterprise/getting-started/install-on-clusters/rke2.mdx]:
```suggestion
- A [compatible RKE2 cluster](../compatibility.mdx#rke2)
```

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx [hunk 4]
```diff
@@ -130,6 +131,58 @@ Monitor progress with the following command:
...
+
+<TabItem label="Rancher UI" value="UI-1">
+
+Unlike the Rancher CLI, the Rancher UI does not provide an option to set the RKE2 CNI value to `none`, which is required by RKE2 to install a non-default CNI like {{prodname}}. To install {{prodname}} using Rancher UI, you must first provision an RKE2 cluster with Calico Open Source, then upgrade it to {{prodname}}.
+
+### Provision RKE2 cluster with Calico Open Source
+
+1. [Provision an RKE2 cluster using Calico as CNI and the default config option](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher)
+2. Validate the RKE2 cluster is set up and running.
+3. Update the Cluster Config by adding the following `HelmChartConfig` to the Additional Manifest section. (In the UI, navigate to **Edit Config**, **Add on config**, **Additional Manifest**.)
```
Comment (sujeet-kr) [inline calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: This is the old step.

File: calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx [hunk 5]
```diff
@@ -2,61 +2,62 @@
...
-The geeky details of what you get:
+   The geeky details of what you get:
 
-<GeekDetails
-  prodname='{{prodname}}'
-  details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes'
-/>
+   <GeekDetails
+    prodname='{{prodname}}'
+    details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:BGP,Datastore:Kubernetes'
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx]:
```suggestion
    details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes'
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx [hunk 6]
```diff
@@ -2,61 +2,62 @@
...
+   />
 
 **Required**
 
-- A [compatible RKE2 cluster](../compatibility.mdx#rke2)
-
+- A [compatible RKE2 cluster](../compatibility.mdx#rke2) 2.6.5 and later
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx]:
```suggestion
- A [compatible RKE2 cluster](../compatibility.mdx#rke2)
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx [hunk 7]
```diff
@@ -130,6 +131,58 @@ Monitor progress with the following command:
...
+
+<TabItem label="Rancher UI" value="UI-1">
+
+Unlike the Rancher CLI, the Rancher UI does not provide an option to set the RKE2 CNI value to `none`, which is required by RKE2 to install a non-default CNI like {{prodname}}. To install {{prodname}} using Rancher UI, you must first provision an RKE2 cluster with Calico Open Source, then upgrade it to {{prodname}}.
+
+### Provision RKE2 cluster with Calico Open Source
+
+1. [Provision an RKE2 cluster using Calico as CNI and the default config option](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher)
+2. Validate the RKE2 cluster is set up and running.
+3. Update the Cluster Config by adding the following `HelmChartConfig` to the Additional Manifest section. (In the UI, navigate to **Edit Config**, **Add on config**, **Additional Manifest**.)
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx]: Old steps

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx (hunk 7 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rke2.mdx]: Fixed.

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx (hunk 4 above)
Comment (sujeet-kr) [reply calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: Not seeing the change.

Comment (sujeet-kr) [inline calico-enterprise/getting-started/install-on-clusters/rke2.mdx]:
```suggestion
```

File: calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rancher-ui.mdx [hunk 8]
```diff
@@ -24,7 +24,7 @@ The geeky details of what you get:
 
 <GeekDetails
   prodname='{{prodname}}'
-  details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes'
+  details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:BGP,Datastore:Kubernetes'
```
Comment (sujeet-kr) [inline calico-enterprise_versioned_docs/version-3.16/getting-started/install-on-clusters/rancher-ui.mdx:27]:
```suggestion
  details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes'
```

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx (hunk 4 above)
Comment (sujeet-kr) [reply calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: Actually on second thought, shouldn't the RancherUI section be removed from RKE2?

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx [hunk 9]
```diff
@@ -2,61 +2,62 @@
 description: Install Calico Enterprise on an RKE2 cluster.
 ---
 
-# RKE2 - Rancher's Next Generation Kubernetes Distribution
+# Rancher Kubernetes Distribution Government (RKE2)
```
Comment (ctauchen) [inline calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: Where does this name come from?

Not seeing it in the docs, at least: https://docs.rke2.io/#why-two-names

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: Removed.

**Review by ctauchen — DISMISSED** [review]
I'd add the CC change also to CC-3.15. Other than that, LGTM

File: calico-enterprise/getting-started/install-on-clusters/rke2.mdx (hunk 9 above)
Comment (bmckercher123) [reply calico-enterprise/getting-started/install-on-clusters/rke2.mdx]: The only change to CC is the version. Rancher UI steps would not work for CC users without backend changes.

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**